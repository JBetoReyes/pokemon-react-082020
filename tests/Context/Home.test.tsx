import UserContext from '@components/hooks/useContext/UserContext.component';
import Home from '@components/hooks/useContext/Home.component';

import React from 'react';
import { mount } from 'enzyme';

describe('AppComponent', () => {
  const user = {
    name: 'test name',
    email: 'test@gmail.com',
  };
  const state = {
    user,
    setUser: jest.fn(),
  };
  const appJsx = (
    <UserContext.Provider value={state}>
      <Home />
    </UserContext.Provider>
  );
  let wrapper = mount(appJsx);
  beforeEach(() => {
    wrapper = mount(appJsx);
  });
  test('Should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
