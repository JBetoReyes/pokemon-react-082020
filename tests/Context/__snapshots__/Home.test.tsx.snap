// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent Should match snapshot 1`] = `
<Home>
  <h1>
    Home component
  </h1>
  <hr />
  {
    "name": "test name",
    "email": "test@gmail.com"
}
</Home>
`;
